### PR TITLE
Trying to resolve Task #10

### PR DESCRIPTION
**Pull request by OjuIsOn**
just to check , I added  a temporary model for blogs as i created the blogs manually in mongo then tried to get them to ensure that everything's alright! and i absolutely have no idea what are those new lines of code in those package-lock.json and i don't know how they got added there automatically , if you guys know why those lines of code got added, i'd love to learn !! .I tried to do it  according to my little understanding of Node js(express), would love to hear and learn from you further!

### CODE REVIEW

**Comment by Debatreya** [comment]
> just to check , I added a temporary model for blogs as i created the blogs manually in mongo then tried to get them to ensure that everything's alright! and i absolutely have no idea what are those new lines of code in those package-lock.json and i don't know how they got added there automatically , if you guys know why those lines of code got added, i'd love to learn !! .I tried to do it according to my little understanding of Node js(express), would love to hear and learn from you further!

Those came as you tried to install the dependencies using `npm`. There was a previous `pnpm-lock` as the repo was first initialised using pnpm.
`pnpm` `yarn` `npm` -> These are basically package managers.
We will not actually care about the `package-lock` file. So no worries.

File: server/src/model/blogs.js
Comment (Debatreya) [inline server/src/model/blogs.js:1]: @OjuIsOn The model you created is not according to the schema. 
Get the schema from `dev-docs.md`

For any other doubt contact your senior on this issue.

File: server/src/routes.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+    app.get("/blogs/get/:id",(req,res)=>{
+        const id_name=(req.params.id);
+        blog.findById(id_name).then((result) => {
+            if(result){
+                res.send(result)
+            }
+        }).catch((err) => {
+            res.send("Something went wrong!Couldn't get the blog ;(")
+        });
+    });
```
Comment (Debatreya) [inline server/src/routes.js:18]: This is not the standard way to write the routes.

Ask your senior She will guide you on the folder structure and the naming convention

**Comment by Debatreya** [comment]
@OjuIsOn you are not supposed to make pull request on this repo.

Check the comments of issue #10 . Ig your senior has shared with you a repo to fork, clone and work on.

And *Tips* : You didn't referenced your issue no. properly.😂. Do it in the Pull Request Description. Like I did above.

**Comment by Debatreya** [comment]
@OjuIsOn dont make pull request in this branch, Currently, we dont have rights to merge your PRs

**Comment by OjuIsOn** [comment]
ookay,,i got it, i am trying my best to make less chaos

**Comment by Debatreya** [comment]
Koi ni. Kro tum code. 😅👏👏

**Comment by Swastik19Nit** [comment]
@OjuIsOn Bhai tum code kar rhe ho yhi badi baat h...keep it up..👏👏

**Comment by Debatreya** [comment]
@Swastik19Nit MVC style nhi hain😅. Edhar `Apna` style hain

**Comment by Debatreya** [comment]
@Swastik19Nit yeh photo kiski chura ke dp lagai!!?😂

**Comment by Swastik19Nit** [comment]
@Debatreya churai nhi hai 😂 generate karwai hai

**Pull request closed without merging**